### ISSUE TITLE
[VULNERABILITY] Update to latest glob 7.0.5

### ISSUE THREAD

**Issue by shazron** [jpillora/node-glob-all#12]
The latest glob 7.0.5 uses minimatch 3.0.2 which fixes a vulnerability.

glob patch:
https://github.com/isaacs/node-glob/issues/268

More info:
https://nodesecurity.io/advisories/118

**Comment by jpillora** [comment]
can you please test the new version works with the examples and send a PR?

**Comment by shazron** [comment]
Unfortunately both tests fail with glob 7.0.5:
```
TAP version 13
# basic
not ok 1 should be equivalent
  ---
    operator: deepEqual
    expected: |-
      [ 'files', 'files/a.txt', 'files/b.txt', 'files/c.txt', 'files/x/z.txt' ]
    actual: |-
      [ 'files', 'files/a.txt', 'files/b.txt', 'files/c.txt', 'files/x', 'files/x/y.txt', 'files/x/z.txt' ]
    at: Test.<anonymous> (/Users/shaz/Documents/Git/Shazron/node-glob-all/test/test.js:30:5)
  ...
not ok 2 should be equivalent
  ---
    operator: deepEqual
    expected: |-
      [ 'files/', 'files/a.txt', 'files/b.txt', 'files/c.txt', 'files/x/z.txt' ]
    actual: |-
      [ 'files/', 'files/a.txt', 'files/b.txt', 'files/c.txt', 'files/x/', 'files/x/y.txt', 'files/x/z.txt' ]
    at: GlobAll.callback (/Users/shaz/Documents/Git/Shazron/node-glob-all/test/test.js:17:7)
  ...
```

If I knew how to fix your lib, I would.

**Comment by mrjonny2** [comment]
Any word on when this is likely to be fixed?

**Comment by jpillora** [comment]
No time atm, though will accept a PR if tests pass :)

**Comment by iandotkelly** [comment]
https://github.com/jpillora/node-glob-all/pull/13 fixes this issue - the two tests continue to pass.

**Comment by jpillora** [comment]
Closed by #13, published as `3.1.0`
